Tasks/InstallSSHKeyV0/postinstallsshkey.ts

```diff
@@ -9,7 +9,12 @@ async function run() {
         let agentPid: string = tl.getTaskVariable(util.postKillAgentSetting);
         if (agentPid) {
             tl.debug('Killing SSH Agent PID: ' + agentPid);
-            ps.kill(+agentPid);
+            try {
+                ps.kill(+agentPid);
+            } catch (err) {
+                // This gets cleaned up by the agent anyways, best effort
```

Comment:
this would also be a good place to explain the problem -- the ssh pid is specifically unreliable

Reply:
https://www.schoonology.com/technology/ssh-agent-windows/